src/modules/organizations/containers/collaborators-container.jsx

```diff
@@ -226,12 +229,26 @@ class CollaboratorsContainer extends React.Component {
   }
 }
 
+CollaboratorsContainer.defaultProps = {
+  location: {},
+  organization: {},
+  organizationCollaborators: [],
+  organizationOwner: {},
+  user: {},
+};
+
 CollaboratorsContainer.propTypes = {
-  children: React.PropTypes.node,
   dispatch: React.PropTypes.func,
+  location: React.PropTypes.shape({
+    query: React.PropTypes.object,
+    page: React.PropTypes.number,
```

Comment:
`page` is an attribute of `query` (so `query` would be `.shape({...` too), and it looks like it might be a number or string?
![screen shot 2017-08-09 at 4 40 43 pm](https://user-images.githubusercontent.com/12118751/29145159-91d750e6-7d21-11e7-9d8a-0b7d3519ee71.png)

can we get away with not defining `page`?